release/pkg/manager/calico/manager.go

```diff
@@ -148,7 +160,14 @@ func releaseImages(version, operatorVersion string) []string {
 	}
 }
 
-func (r *ReleaseBuilder) Build() error {
+func (r *CalicoManager) helmChartVersion() string {
+	if r.chartVersion == "" {
```

Comment:
We need comments explaining that this is not used in OSS if we're going to include logic that isn't valid for OSS.

Reply:
Otherwise, this should just be `return r.calicoVersion`